tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -210,27 +210,6 @@ trait WskWebActionsTests
             authorizedResponse.body.asString shouldBe namespace
     }
 
-    if (testRoutePath == "/api/v1/web") {
```

Comment:
If you want to replace it with a unit test then you'll need to instatiate the top level route in the controller.